src/gui/src/layoutViewer.cpp

```diff
@@ -3032,6 +3037,14 @@ void LayoutViewer::addFocusNet(odb::dbNet* net)
   }
 }
 
+void LayoutViewer::addRouteGuides(odb::dbNet* net)
+{
+  const auto& [itr, inserted] = route_guides_.insert(net);
+  if (inserted) {
+    fullRepaint();
+  }
```

Comment:
This should be handled in the renderer you added, otherwise there is no point in having it.